scripts/projectView.js

```diff
@@ -0,0 +1,52 @@
+(function(module){
+
+  var projectView = {};
+
+  projectView.handleMainNav = function() {
+    $('nav').on('click', '.tab', function(e) {
+      $('.tab-content').hide();
+      var $selectedTab = $(this).attr('data-content');
+      $('#' + $selectedTab).fadeIn(500);
+    });
+    $('nav .tab:nth-of-type(2)').click();
+  };
+  projectView.populateFilters = function() {
+    $('div[data-category]').each(function() {
```

Comment:
You don't have anything matching div[data-category]. I'd suggest trying $('article h1').text() instead.